### PR TITLE
More magwells

### PR DESCRIPTION
**Pull request by Drofseh**
**When merged this pull request will:**
- Change 570x28 to 57x28, since the cartridge is always called the 5.7x28mm
This is more correct and keeps the naming convention of other magwells that do not add trailing zeros.
- Add magwels for the CZ 75, 85, 97, 2075, P-01, P-06, P-07, P-09 for Urban.
- Add .45 ACP fullsize Glock magwell to SMG_01 (Vermin SMG/KRISS Vector)
- Add 9x19mm Scorpion Evo 3 magwell for SMG_02 (Sting 9mm/CZ Skorpion Evo)
- Add 9x19mm MP5 magwell to SMG_05 (Protector 9 mm/MP5K)
* I'm not 100% happy with having these vanilla two vanilla SMGs as 9x19mm since in vanilla they use 9x21mm and because they should not be able to interchange magazines with each other IRL.
However I think it would be confusing for players if weapons that appear to be able to interchange magazines with mod weapons aren't able to (for example the vanilla MP5K and the NIA or CUP MP5's).

### CODE REVIEW

**Comment by PabstMirror** [comment]
changing name from #1061 which hasn't been released, so no compat problems

File: addons/jam/CfgWeapons.hpp [hunk 1]
```diff
@@ -101,7 +101,20 @@ class CfgWeapons {
         magazineWell[] = {"CBA_338NM_LINKS"};
     };
 
-    class SMG_03_TR_BASE: Rifle_Base_F {
-        magazineWell[] = {"CBA_570x28_P90"};
+    class SMG_01_Base : Rifle_Short_Base_F {
```
Comment (commy2) [inline addons/jam/CfgWeapons.hpp]:
```suggestion
    class SMG_01_Base: Rifle_Short_Base_F {
```

File: addons/jam/CfgWeapons.hpp [hunk 2]
```diff
@@ -101,7 +101,20 @@ class CfgWeapons {
...
+        magazineWell[] = {"CBA_45ACP_Glock_Full"};
     };
+
+    class SMG_02_base_F : Rifle_Short_Base_F {
```
Comment (commy2) [inline addons/jam/CfgWeapons.hpp]:
```suggestion
    class SMG_02_base_F: Rifle_Short_Base_F {
```

File: addons/jam/CfgWeapons.hpp [hunk 3]
```diff
@@ -101,7 +101,20 @@ class CfgWeapons {
...
+        magazineWell[] = {"CBA_9x19_ScorpionEvo3"};
+    };
+
+    class SMG_03_TR_BASE : Rifle_Base_F {
```
Comment (commy2) [inline addons/jam/CfgWeapons.hpp]:
```suggestion
    class SMG_03_TR_BASE: Rifle_Base_F {
```

File: addons/jam/CfgWeapons.hpp [hunk 4]
```diff
@@ -101,7 +101,20 @@ class CfgWeapons {
...
+        magazineWell[] = {"CBA_57x28_FNP90"};
+    };
+
+    class SMG_05_base_F : Rifle_Short_Base_F {
```
Comment (commy2) [inline addons/jam/CfgWeapons.hpp]:
```suggestion
    class SMG_05_base_F: Rifle_Short_Base_F {
```

File: addons/jam/CfgMagazineWells.hpp [hunk 5]
```diff
@@ -32,7 +32,7 @@ class CfgMagazineWells {
 
     // Pistol calibre magwells, ordered lexicographically in metric and imperial groups
     #include "magwells_10mmAuto.hpp"    // 10mm Auto | 10mm Automatic | 10x25mm
-    #include "magwells_570x28.hpp"      // 5.70x28mm
+    #include "magwells_57x28.hpp"       // 5.7x28mm
```
Comment (commy2) [inline addons/jam/CfgMagazineWells.hpp]: This one is from 3.9 or earlier, right?
```suggestion
    #include "magwells_570x28.hpp"      // 5.7x28mm
```
Have to keep it for bwc.

File: addons/jam/magwells_57x28.hpp [hunk 6]
```diff
@@ -0,0 +1,6 @@
+    class CBA_57x28_FN57 {};        // FN Five-Seven
```
Comment (commy2) [inline addons/jam/magwells_57x28.hpp]: Renamed. See above.

File: addons/jam/CfgMagazineWells.hpp (hunk 5 above)
Comment (jonpas) [reply addons/jam/CfgMagazineWells.hpp]: No, it's from #1061.

Comment (commy2) [reply addons/jam/CfgMagazineWells.hpp]: Then I support changing this now.

**Comment by Drofseh** [comment]
@commy2 requested changes done.

**Pull request merged**